tests/unit/downloader/test_flattener.py

```diff
@@ -1,7 +1,7 @@
 import json
```

Comment:
this file is too big. Consider splitting by:

- split data and test logic by removing literal json data to files
- splitting  to suites by features tested

Reply:
@aaclan-ebi are you going to split this file?

Reply:
yup, still thinking how...

Reply:
wip

Reply:
Done!  I grouped the test cases and had a test file/suite for each grouping:
1. test_flattener - contains **generic** logic for flattening **fields**
2. test_flatten_ontology_module - contains logic for flattening **ontology module objects**
3. test_flatten_project_module - contains logic for flattening **project modules**

I also extracted big json's into their own files. However, there are some parts of the JSON that I want to emphasize/explicitly show in some test cases which I believe makes it easier to see and  better understand the logic.